tests/integration/components/frost-object-browser-test.js

```diff
@@ -59,33 +55,17 @@ describe(test.label, function () {
       expect(this.$('.frost-object-browser-facets-header')).to.have.text('Refine by')
     })
 
-    it('with filterVisible=true, should display facets-expanded-view', function () {
+    it('without isFilterHiddenOnLoad, should display facets-expanded-view', function () {
       expect($hook('myObjectBrowser-facets-expanded-view').is(':visible')).to.eql(true)
     })
 
-    it('with filterVisible=true, should hide facets-collapsed-view', function () {
+    it('without isFilterHiddenOnLoad, should hide facets-collapsed-view', function () {
```

Comment:
Between these two tests it should be tested what happens when `isFilterHiddenOnLoad` is to `true` and `false`.  Right now both tests are testing the same thing.  Update the test description to reflect the proper states as well.

Reply:
How are these tests different than are what begin on line 100?

Reply:
`isFilterHiddenOnLoad` is set to false in one test and skipped in another.

Reply:
Is this answer in response to my first inquiry or the one about line 100?